WordPress/gutenberg
# Block: social links.

## Description

Fixes: https://github.com/WordPress/gutenberg/issues/1873

## References
[slack discussion](https://wordpress.slack.com/archives/C02QB2JS7/p1553901163077800)
[Prototype](https://www.figma.com/proto/gql7fzWm5Ch401zkrLnBgvqU/social-icons-block-i2?node-id=0%3A2040&scaling=min-zoom)

## How has this been tested?

- Add new Social Links block
- Confirm you can insert multiple links
- Icons and links show on view
- Can create and use block as Author permission

## Screenshots 

(see updates in thread)


## Types of changes

Adds a new block for inserting social media links. The block will try to match the proper icon to the URL typed in, or the user can select the icon by clicking and selecting from a list.


## Checklist:
